### DISCUSSION TITLE
Non-use of “Index” as a term / label

### DISCUSSION THREAD

**Discussion by MikeHypercube** [edmcouncil/fibo#1353] [General]
Add as synonym as appropriate (no synonyms seen in this model – check EA model for synonyms).

**Comment by rivettp** [comment]
We have "indices" as part of the name of one of our major domains but don't seem to actually define it.

**Comment by ElisaKendall** [comment]
We define economic indicator but not indicator, and we define reference index - of which the plural is indices - what are you looking for here?

**Comment by trypuz** [comment]
@MikeHypercube, what is this issue about? Could you explain the problem, please?

**Comment by MikeHypercube** [comment]
The original issue that I had anything to do with was closed in 2018. It has been reopened by Pete Rivett apparently because standard industry usage of certain words for a general category of thing is not self-explanatory enough. Please address any further questions to Pete.

**Comment by rivettp** [comment]
My point was that we have a whole domain called "Indices and Indicators" but we don't define the general concept of Index (or Indicator), just single specializations Economic Indicator and Reference Index.
